client/src/views/logs.tsx

```diff
@@ -115,10 +148,20 @@ export default class Logs extends Base<Props, State> {
                         <div className='logs_showPreviousLabel'>Previous</div>
                     </label>
 
+
                     <InputFilter
                         filter={filter}
                         onChange={x => this.setState({filter: x})}
                     />
+
+                    <Button
+                        className="button_clear log_downloadButton"
+                        title="Download Log"
+                        disabled={logDownloading}
```

Comment:
I think this is a fine starting point. It would be nice if we could display some sorta "working" indicator though. I'd be curious how it looks if instead of disabling the button, we replaced it with `<Loading />` (which is a shared component we use throughout the app).

Reply:
Changed to show LoadingSvg instead of disabling the button